ledger-service/http-json-perf/src/main/scala/com/daml/http/perf/Main.scala

```diff
@@ -323,4 +90,87 @@ object Main extends StrictLogging {
     }
     simulationLog.map(_ => ())
   }
+
+  def main(args: Array[String]): Unit = {
+    val name = "http-json-perf"
+    val terminationTimeout: FiniteDuration = 30.seconds
+
+    implicit val asys: ActorSystem = ActorSystem(name)
+    implicit val mat: Materializer = Materializer(asys)
+    implicit val aesf: ExecutionSequencerFactory =
+      new AkkaExecutionSequencerPool(poolName = name, terminationTimeout = terminationTimeout)
+    implicit val elg: EventLoopGroup = Transports.newEventLoopGroup(true, 0, "gatling")
+    implicit val ec: ExecutionContext = asys.dispatcher
+
+    def terminate(): Unit = {
```

Comment:
I would prefer not to use discard here, everything is side effecting, it's obvious. If fatal warnings can;t be removed maybe just suppress the warning inside terminate.